test/common/testlib.py

```diff
@@ -631,8 +631,7 @@ def login_and_go(self, path=None, user=None, host=None, authorized=True):
 
         # HACK https://bugzilla.redhat.com/show_bug.cgi?id=1461893
         # selinux errors while logging in via ssh
-        'type=1401 audit(.*): op=security_compute_av reason=bounds scontext=system_u:system_r:sshd_t:s0-s0:c0.c1023 tcontext=unconfined_u:system_r:svirt_lxc_net_t:s0-s0:c0.c1023 tclass=process perms=transition,sigchld,sigstop,signull,signal,getattr',
-        'type=1401 audit(.*): op=security_compute_av reason=bounds scontext=system_u:system_r:sshd_t:s0-s0:c0.c1023 tcontext=unconfined_u:unconfined_r:svirt_lxc_net_t:s0-s0:c0.c1023 tclass=process perms=transition,sigchld,sigstop,signull,signal,getattr',
+        'type=1401 audit(.*): op=security_compute_av reason=bounds .* tclass=process perms=transition.*',
```

Comment:
Different distro? RHEL Atomic is built from RHEL 7 packages, so it's the same packages (container-selinux-2.12-2.gite7096ce.el7.noarch and selinux-policy-targeted-3.13.1-102.el7_3.16.noarch).

Also, I don't think we need to be overly strict here -- this is *not* a denial, it's just an audit message (of which there are plenty already), and it's not actually breaking functionality.

If you insist, I'll file (yet another) new bug and add more match pattern, but TBH it seems like maintenance busywork to me.